app/build.gradle

```diff
@@ -25,7 +25,7 @@ android {
         targetSdkVersion 28
         versionCode 11 // This versionCode is "frozen" for local builds. For "release" builds we
         // override this with a generated versionCode at build time.
-        versionName "3.2.2"
+        versionName "3.2.6.1"
```

Comment:
I'm not sure if we should just bump per major/minor version? Or have a commit for all releases. In any case, when we do the 3.3 release, I'll update this version too, but we need the CHANGELOG changes.

Reply:
Not sure I'm understanding you correctly, I do think we need to tag everything that goes out to users differently though.  There will be a lot of headings under our releases tab, but I think it is important to keep that record.